File: problems/2068_check_whether_two_strings_are_almost_equivalent/solution.cpp
#include <vector>
#include <string>
#include <unordered_map>
#include <unordered_set>
#include <algorithm>
#include <climits>
#include <queue>
#include <stack>
#include <cmath>
#include <numeric>
#include <set>
#include <map>
using namespace std;

class Solution {
public:
    bool checkAlmostEquivalent(string word1, string word2) {
        int freq[26] = {};
        for (char c : word1) freq[c - 'a']++;
        for (char c : word2) freq[c - 'a']--;
        for (int i = 0; i < 26; i++) {
            if (abs(freq[i]) > 3) return false;
        }
        return true;
    }
};

File: problems/2068_check_whether_two_strings_are_almost_equivalent/solution.py
class Solution:
    def checkAlmostEquivalent(self, word1: str, word2: str) -> bool:
        from collections import Counter
        c1 = Counter(word1)
        c2 = Counter(word2)
        for c in set(word1 + word2):
            if abs(c1[c] - c2[c]) > 3:
                return False
        return True
